ht16/libs/SeleniumCustom.py

```diff
@@ -0,0 +1,12 @@
+from selenium.webdriver.support import expected_conditions as ec
+from selenium.webdriver.support.wait import WebDriverWait
+from robot.libraries.BuiltIn import BuiltIn
+from robot.utils import timestr_to_secs
+
+
+class SeleniumCustom:
+
+    def wait_until_element_is_stale(self, web_element, timeout, poll_interval):
```

Comment:
It checks if the delete action was performed. 
In case there were multiple rows of products in the cart there would be always an element for the same first row locator. Checking if the list was refreshed after the delete action and the element is now stale was the easiest solution.
Haven't found it in a SeleniumLibrary, so get one from Selenium provided condition.
Plus was interesting to understand how to get running driver from a library.